src/patternfly/components/DataList/data-list.scss

```diff
@@ -0,0 +1,207 @@
+@import "../../patternfly-utilities";
+
+.pf-c-data-list {
+  --pf-c-data-list--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
```

Comment:
Should this be `--pf-c-data-list--BackgroundColor: transparent`, as we decided that `pf-c-card` would provide background and box-shadow?

Reply:
we'll leave it as is for now and address later once we figure out theming